mojave/effects/spawners/lootdrop/gunloot.dm

```diff
@@ -22,7 +22,6 @@
 	var/loot4 = list(
 				/obj/item/gun/ballistic/automatic/pistol/ms13/pistol22,
 				/obj/item/ammo_box/magazine/ms13/m22,
-				""
```

Comment:
Keep the quotes, this applies to the entire file. The gun loot spawner checks for three items, the extra empty spots are filled in by the quotes to minimize potential issues.